src/pixbuf/mod.rs

```diff
@@ -181,6 +183,66 @@ impl Pixbuf {
         }
     }
 
+    /// Create a new GdkPixbuf containing a copy of the current pixbuf scaled
+    /// to `dest_width` x `dest_height`. The calling pixbuf remains unaffected.
+    /// `interp_type` should be `Nearest` if you want maximum speed (but when
+    /// scaling down `Nearest` is usually unusably ugly). The default
+    /// `interp_type` should be `Bilinear` which offers reasonable quality and
+    /// speed.
+    ///
+    /// You can scale a sub-portion of `src` by creating a sub-pixbuf pointing
+    /// into src ; see `new_subpixbuf()`.
+    ///
+    /// For more complicated scaling/compositing see `scale()` and
+    /// `composite()`.
+    pub fn scale_simple(&self, dest_width: i32, dest_height: i32, interp_type: InterpType) -> Option<Pixbuf> {
```

Comment:
I believe we've settled on using `Result` for failures. At least that's what `new` does.
We could rethink this particular failure mode and just panic on the OOM failures.